runtime/doc/api.txt

```diff
@@ -1584,7 +1584,7 @@ nvim_set_keymap({mode}, {lhs}, {rhs}, {*opts})             *nvim_set_keymap()*
                     {rhs}   Right-hand-side |{rhs}| of the mapping.
                     {opts}  Optional parameters map. Accepts all
                             |:map-arguments| as keys excluding |<buffer>| but
-                            including |noremap| and "desc". |desc| can be used
+                            including |noremap| and "desc". "desc" can be used
```

Comment:
I was actually planning on adding `<desc>` to `:map` commands as regular map-argument.

Reply:
You haven't added it yet. This reference is doesn't currently mske sense. You can change it again later.

Reply:
Ok makes sense :)